src/org/usfirst/frc2175/subsystem/RobotSubsystems.java

```diff
@@ -12,7 +12,7 @@
     private final DriverStation driverStation;
 
     // TODO add remaining subsystems
-    private final DreamIntakeSubsystem dreamIntakeSubsystem;
+    public final DreamIntakeSubsystem dreamIntakeSubsystem;
```

Comment:
You guys need to learn to ignore Eclipse's suggestions! :smile: 

We keep these variables private, but we have public getter methods so that other things can access these variables. See my other comment...